src/labels/CustomLabels.labels

```diff
@@ -2752,6 +2752,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>- Household Duplicate</value>
     </labels>
     <labels>
+        <fullName>hhErrorInvalidContactFieldName</fullName>
+        <categories>Households, Household Naming</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>hhErrorInvalidContactFieldName</shortDescription>
+        <value>The field, {0}, does not match any field name on Contact.</value>
```

Comment:
This error can get displayed in places not obviously related to the settings page, such as creating a new contact, if they already have invalid formats saved.  Here's what the message is when trying to save a new contact (note I bold italicized the part we control):

Error: Invalid Data. 
Review all error messages below to correct your data.
Apex trigger TDTM_Contact caused an unexpected exception, contact your administrator: TDTM_Contact: execution of BeforeInsert caused by: HH_HouseholdNaming.HouseholdNamingException: _**The field, FirstName}, does not match any field name on Contact.**_: Class.HH_HouseholdNaming: line 259, column 1

I suggest that we add more context to the error so it says something like:

Invalid {0}: The field,  FirstName}, does not match any field name on Contact.

where we'd replace {0} with Household Name Format, Formal Greeting Format, or Informal Greeting Format.  this way the user would have more context on how to resolve the problem.